17/debian/bookworm-slim/hotspot/Dockerfile

```
@@ -1,6 +1,6 @@
-ARG JAVA_VERSION="17.0.8.1_1"
 ARG BOOKWORM_TAG=20230904
-FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-jammy as jre-build
+ARG JAVA_VERSION=17.0.8.1_1
+FROM eclipse-temurin:"$JAVA_VERSION"-jdk-jammy as jre-build
```

```suggestion
FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-jammy as jre-build
```

To stay coherent with the Dockerfile already using an `ARG` (and it is a good practise)